test/unit/org/apache/cassandra/cql3/CQL3TypeLiteralTest.java

```diff
@@ -1,760 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.cassandra.cql3;
-
-import java.math.BigDecimal;
-import java.math.BigInteger;
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-import java.nio.ByteBuffer;
-import java.util.*;
-import java.util.regex.Pattern;
-
-import org.junit.Test;
-
-import org.apache.cassandra.db.marshal.*;
-import org.apache.cassandra.serializers.*;
-import org.apache.cassandra.utils.TimeUUID;
-import org.apache.cassandra.transport.ProtocolVersion;
-import org.apache.cassandra.utils.ByteBufferUtil;
-
-import static org.apache.cassandra.utils.Clock.Global.currentTimeMillis;
-import static org.junit.Assert.assertEquals;
-
-/**
- * Test functionality to re-create a CQL literal from its serialized representation.
- * This test uses some randomness to generate the values and nested structures (collections,tuples,UDTs).
- */
-public class CQL3TypeLiteralTest
```

Comment:
deleted this test as its a duplicate of `AbstractTypeTest.serde`. This test actually showed we were doing the wrong thing, it built its own model of what CQL would accept and made sure types matched its model; but if you ask CQL to parse it will fail as the model wasn't correct... Rather than fixing, was easier to remove as the new `serde` test covers this completely.